plugins/webp-uploads/fallback.js

```diff
@@ -1,8 +1,11 @@
-/* eslint no-var: "off", camelcase: "off" */
+/* eslint no-var: "off", camelcase: "off", no-undef: "off" */
```

Comment:
I believe this change can now be reverted?

```suggestion
/* eslint no-var: "off", camelcase: "off" */
```